### ISSUE TITLE
Port help

### ISSUE THREAD

**Issue by sameethan1** [joshcamas/unity3d-billboard-generator#1]
Hi, I know nothing about what this C# says, but I read that it can be ported to Builtin which is what I use...
I'm just wondering if you could help me out, cause this has been the only thing closest to creating billboards that I'd love to use, please and Thank You!

Farthest I got with guessing was changing UnityEngine.Rendering.Universal to just UnityEngine.Rendering and that fixed the main console errors, but then more lines needed changing, so trying to learn and understand but I have no idea xD
I just build VRChat worlds and use Graphs for shaders/scripts for the most part...

**Comment by HunterAhlquist** [comment]
Yup, having the same issue, it appears this requires the Universal Rendering pipeline which I can't use for my project.
